components/collection/CollectionGrid.vue

```diff
@@ -198,6 +198,16 @@ watch(
     resetPage()
   }
 )
+
+watch(
+  collections,
+  (cols) => {
+    emit('count', cols.length)
```

Comment:
It's not the total amount of the collection. Please check `totalCount` in the graphql  query`collectionListWithSearch`

Reply:
Any update on this?

Reply:
```suggestion

```